V 3 / Add wui-transaction-visual

svenvoskamp opened the pull request:
# Changes

- feat: Add `wui-transaction-visual` component

0xAsimetriq commented on packages/ui/src/composites/wui-transaction-visual/index.ts:
```diff
@@ -0,0 +1,54 @@
+
+@customElement('wui-transaction-visual')
+export class WuiTransactionVisual extends LitElement {
+  public static styles = [resetStyles, elementStyles, styles]
```
Are `elementStyles` needed here, since no form elements are used?

0xAsimetriq commented on packages/ui/src/composites/wui-transaction-visual/index.ts:
```diff
@@ -0,0 +1,54 @@
+  public render() {
+    let color: 'blue-100' | 'error-100' | 'success-100' = 'blue-100'
+    let icon: TransactionIconType = 'arrowTop'
+    if (
```
You can make small arrays here i.e `const outgoing = ['bought', 'buy', 'deposited']` etc. and then use them in if statements like `if (outgoing.includes(this.type))`, should help simplify thinsg

svenvoskamp commented:
* Simplified statements in `wui-transaction-visual`
* Removed elementStyles

I will let the `imageSrc` be for what is right now as we will postpone this for a later version.
